drag_and_drop_v2/locale

```diff
@@ -1 +1 @@
-translations
\ No newline at end of file
+conf/locale
```

Comment:
Could you please add some more information to the PR's description? Currently, it's unclear why we want to make these changes.

1. What should we do about the existing `translations` directory?
2. Will the translations be updated automatically with some external automation from now on? 
3. What is the standardized structure of the translation directories? E.g., `ora2` has translations in the [openassessment/locale](https://github.com/openedx/edx-ora2/tree/51fe75f/openassessment/locale) directory and a `../openassessment/locale` link in [conf/locale](https://github.com/openedx/edx-ora2/blob/4d350ac6aadb5df797dc2577e8a1db45faf90ea5/conf/locale).

Reply:
1- `translations` directory will be deleted by the end of the project

2- once this PR and the related one in `open-translations` are merged, this directory will be useless. (I'm assuming that Atlas is working fine with it, it is something we're going to test and monitor)

3- We're standardizing the pattern used for all repos. Python code like `edx-platform` uses `conf/locale`, so it makes sense to make that the default for all XBlocks too. See [OEP-58](https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0058-arch-translations-management.html) for all the needed details

Reply:
@shadinaif, thanks for clarifying these points! Would you mind adding them to the PR's description (instead of keeping the default template) for better discoverability?

Reply:
Marking this as resolved since the functionality introduced by the PR has changed. Please check the PR description